cmd/ipfs/runmain_test.go

```diff
@@ -7,16 +7,15 @@ import (
 	"fmt"
 	"io/ioutil"
 	"os"
-	"testing"
 )
 
 // this abuses go so much that I felt dirty writing this code
 // but it is the only way to do it without writing custom compiler that would
 // be a clone of go-build with go-test
-func TestRunMain(t *testing.T) {
+func (m mainRun) TestRunMain(c *gc.C) {
```

Comment:
Not sure if it will fly with coverage collection code. We will see when dependency thing is fixed.